ws-nextjs-app/pages/[service]/send/[id]/FormField/TextArea.tsx

```diff
@@ -14,10 +14,11 @@ export default ({
   hasAttemptedSubmit,
 }: InputProps) => {
   const { isValid, value = '', required, wasInvalid } = inputState;
-
+  const translation = 'Maximum 500 Words';
   return (
     <>
       <Label id={id}>{label}</Label>
+      <p css={styles.textAreaLabel}>{translation}</p>
```

Comment:
Could use the `Paragraph` component here and remove the font styling class. You can pass in `fontVariant` and size` into the component to define the font styling that way.